examples/minitest_helper.rb

```diff
@@ -35,4 +35,5 @@ def use_natural_assertions_if_supported(enabled=true)
 end
 
 Minitest::Spec.send(:include, GivenAssertions)
+Minitest::Test.send(:include, GivenAssertions)
```

Comment:
I take back this comment I didn't realize it was just mixing in Jim's own test-scoped assertions. Those assertions are only useful for asserting Given's implementation.